.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "bootstrap"]
 	path = bootstrap
-	url = git@github.com:kubicorn/bootstrap.git
+	url = https://github.com:kubicorn/bootstrap.git
```

Comment:
Shouldn't this be `https://github.com/...` (`/` instead of `:`)?

Reply:
well spotted 👍